docs/architecture/cloud-native/toc.yml

```diff
@@ -44,7 +44,7 @@
     - name: Service Mesh communication infrastructure
       href: service-mesh-communication-infrastructure.md
   - name: Distributed Data
-    href: 
+    href: database-per-microservice.md
```

Comment:
This entry is already in the line 50. You can just remove this.

```suggestion
```

Reply:
Good catch, removed that in the latest update of the TOC. The PDF generator had added the content twice. Fixed.

Reply:
Can you apply my suggestion here?

Reply:
Yes, in the updated toc.yml file, you should see...

  - name: Distributed Data
    href: database-per-microservice.md
    items:
    - name: Relational vs. NoSQL data
      href: relational-vs-nosql-data.md
    - name: Caching in a cloud-native application
      href: azure-caching.md
    - name: Elasticsearch in Azure
      href: elastic-search-in-azure.md

Reply:
I still feel you should keep lines 49 and 50 and remove this one.

This is what you have now:
![image](https://user-images.githubusercontent.com/12971179/78067878-5476c500-734c-11ea-8354-d119f9b619c8.png)

Adding the link to the TOC top level node, adds a duplicate node under

Reply:
Done

Reply:
@mairaw Is this change correct? I approved the changes by verifying the review site. However, I am not clear about what was wrong in the first place.

@robvet The title, file name, and the links say different things. Can we fix the title and the file name to be Distributed Data(distributed-data.md) itself and have sub-title that is "Database per microservice"? (Maybe in a separate PR)

Reply:
Spoke with Maira - We will address this in a different PR.